esx_service/vmci/vmci_client.c

```diff
@@ -348,6 +349,15 @@ Vmci_GetReply(int port, const char* json_request, const char* be_name,
 
    req.mlen = strnlen(json_request, MAXBUF) + 1;
    req.msg = json_request;
+   ans->buf = NULL;
```

Comment:
There are error paths in the vsock_getreply() function that return a valid error code ahead of allocating the receive buffer. Yes, its just initializing the field. Can allocate the be_answer structure with C.calloc() vs. C.malloc() perhaps.